core/modules/system/system.install

```diff
@@ -3653,6 +3656,58 @@ function system_update_1098() {
   }
 }
 
+/**
+ * Set the 'initial version' based on the system install time.
+ */
+function system_update_1099() {
+  $config = config('system.core');
+  $install_version = $config->get('install_version');
+  if (!isset($install_version)) {
+    $install_time = state_get('install_time');
+    $version_timestamps = [
```

Comment:
Install version could be useful for Telemetry.

Reply:
@herbdool  I would not have thought of this, but I can also imagine that now that we have this setting other contrib projects might find useful ways of making use knowing the install version of old sites. I'm not sure what that use would be, but I understand why it might be valuable and why now might be a good time to do the update.

I have no strong perference either way. This was in the PR when I took it over.

Reply:
This was discussed in the issue. I had originally used `BACKDROP_VERSION` here but there was pushback so the list was added :)